scripts/s390/initial-setup.csh

```diff
@@ -7,6 +7,6 @@ set IS_UNIT = initial-setup.service
 if ( { systemctl -q is-enabled $IS_UNIT } && -x $IS_EXEC ) then
     # check if we're not on 3270 terminal and root
     if (( `/sbin/consoletype` == "pty" ) && ( `/usr/bin/id -u` == 0 )) then
-        $IS_EXEC --no-stdout-log && systemctl -q is-enabled $IS_UNIT && systemctl -q disable $IS_UNIT
+        $IS_EXEC --no-stdout-log --no-multi-tty && systemctl -q is-enabled $IS_UNIT && systemctl -q disable $IS_UNIT && systemctl -q stop $IS_UNIT
```

Comment:
Am I wrong or the ``systemctl -q disable $IS_UNIT && systemctl -q stop $IS_UNIT`` is fixing other bug? If yes, could you please create a separate commit with explanation?

Reply:
Good point - I've split the change into to PRs, one adding the CLI option and another one changing the s390 scripts and documenting via the commit message why.